modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/IgniteDataStorageMetricsSelfTest.java

```diff
@@ -423,6 +477,61 @@ public void testWalCompressedBytes() throws Exception {
         assertCorrectWalCompressedBytesMetrics(n1);
     }
 
+    /**
+     * Populates a cache w/32 KB of data
```

Comment:
Please fix javadoc by https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-JavadocComments

Reply:
Forgot the dot at the end.

Reply:
Not fixed.